warehouse/query-core/src/main/java/datawave/query/jexl/JexlNodeFactory.java

```diff
@@ -225,7 +225,7 @@ else if (1 == valuesForField.size()) {
                     childNode.jjtSetParent(wrappedChildNode);
```

Comment:
An initial test I made

```java
   @Test
    public void testNoFieldExpansion() throws ParseException {
        IndexLookupMap indexLookupMap = new IndexLookupMap(10, 10);
        indexLookupMap.putAll("FOO", Sets.newHashSet("6", "9"));
        indexLookupMap.putAll("BAR", Sets.newHashSet("7", "8"));
    
        JexlNode originalNode = JexlASTHelper.parseJexlQuery("((_Bounded_ = true) && (FOO > 5 && FOO < 10))");
        JexlNode node = JexlNodeFactory.createNodeTreeFromFieldsToValues(JexlNodeFactory.ContainerType.OR_NODE, new ASTEQNode(ParserTreeConstants.JJTEQNODE),
                        originalNode, indexLookupMap, false, true, false);
        
        assertEquals("(FOO == '6' || FOO == '7' || FOO == '8' || FOO == '9')", JexlStringBuildingVisitor.buildQuery(node));
        assertTrue(JexlASTHelper.validateLineage(node, false));
    }
```
will still fail unless the line `childNode.jjtSetParent(wrappedChildNode);` here is deleted. Effectively, this line is setting the parent to the reference rather than the correct referenceexpression.

Reply:
Good catch, I'll take another pass through this